docs/users/settings-appearance.md

```diff
@@ -116,7 +116,11 @@ You can customize the appearance of the website and app.
 <Tabs groupId="os">
   <TabItem value="web" label="Website">
 
-    This feature is not available on the website.. 
+    * Visit [GeyserTimes](https://geysertimes.org) with your browser.
+    * Log in to your account.
+    * In the **Profile & Settings** card, click **Choose Selected Geysers...**
+    * Type the names of up to 15 geysers.
+    * The time elapsed since the last recorded eruption of each selected geyser will display on the GeyserTimes homepage.
```

Comment:
I would suggest showing this information in as a tip similar to the Android one to make it consistent.